src/sentry/conf/types/consumer_definition.py

```diff
@@ -5,11 +5,19 @@
 
 import click
 
+from sentry.conf.types.topic_definition import Topic
+
 
 class ConsumerDefinition(TypedDict, total=False):
-    # Which logical topic from settings to use.
-    topic: Required[str | Callable[[], str]]
-    default_topic: str
```

Comment:
`default_topic` is replaced by `validate_schema` since it was only actually controlling whether schema validation was run or not